Servlets/web/index.html

```diff
@@ -0,0 +1,16 @@
+<%--
```

Comment:
<% - is jsp stuff but it is index.**html**
so Jasper will not take place here.
To use jsp stuff it needs to have .jsp extension